lib/Conversion/TorchToTcp/Elementwise.cpp

```diff
@@ -105,6 +105,72 @@ class ConvertAtenMulOp : public OpConversionPattern<AtenMulTensorOp> {
   }
 };
 
+class ConvertAtenBatchNormOp : public OpConversionPattern<AtenBatchNormOp> {
+public:
+  using OpConversionPattern<AtenBatchNormOp>::OpConversionPattern;
+  using OpAdaptor = typename AtenBatchNormOp::Adaptor;
+
+  LogicalResult
+  matchAndRewrite(AtenBatchNormOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    Value input = adaptor.getInput();
+    RankedTensorType inputType = input.getType().dyn_cast<RankedTensorType>();
+
+    Value weight = adaptor.getWeight();
+    RankedTensorType weightType = weight.getType().dyn_cast<RankedTensorType>();
+
+    Value bias = adaptor.getBias();
+    RankedTensorType biasType = bias.getType().dyn_cast<RankedTensorType>();
+
+    Value running_mean = adaptor.getRunningMean();
```

Comment:
I believe MLIR coding convention is `runningMean`.  Same elsewhere.

Reply:
Fixed.